server/src/inmantals/server.py

```diff
@@ -195,7 +220,7 @@ def setup_project():
                     else:
                         module.Project.set(module.Project(project_dir))
                 else:
-                    module.Project.set(module.Project(project_dir))
+                    module.Project.set(module.Project(project_dir, attach_cf_cache=useCache))
```

Comment:
@sanderr is this ok? 
I don't really like this part
`os.getenv("ENV_VAR", "False").lower() in ("true", "1")` but is seems I always get a string from os.getenv